### ISSUE TITLE
Reduce Redundancy in your templates

### ISSUE THREAD

**Issue by 2h311** [berrysauce/whatdevsneed#14]
I want to ask why you didn't use Jinja's shared templating to avoid repeating code in your templating.

**Comment by berrysauce** [comment]
As you might have seen in the commits, I haven't updated the actual code of whatdevsneed recently – I've just updated it to keep it running. It needs a design & code rework (from the ground up). I'll keep your suggestion in my mind! 😀

**Comment by 2h311** [comment]
Yes, it surely does need rework. I'm currently looking for project ideas, and I stumbled upon your Github. I like your eye for UI, do you work with a UI designer?

**Comment by berrysauce** [comment]
No, all self-made and self-taught.

**Comment by 2h311** [comment]
And how do you come up with your project ideas? I'm looking to build stuff to better my knowledge of Python and coding in general.

**Comment by berrysauce** [comment]
I don't know, you need to find a niche you're interested in or just make something that's better than the already existing products. Or just find something that already exists and just build your own to practice & learn. That's usually how I get started on projects.

**Comment by 2h311** [comment]
Interesting, nice talking to you.

**Comment by berrysauce** [comment]
Sure! Nice talking to you too! You can keep this open so I don't forget your recommendation in the future 😊

**Comment by 2h311** [comment]
Not a problem, Paul. Hope you don't mind me checking up on other interesting projects you have, lol.
